### DISCUSSION TITLE
Prestashop Checkout  v8.3.3.0 does not check credit card field

### DISCUSSION THREAD

**Discussion by jlores** [PrestaShop/PrestaShop#33031] [Q&A]
Hello,

Yesterday the version 8.3.3.0 of prestashop checkout module came out, today carrying out tests, it allows you to press the confirm order button without anything in the credit card field. 

I understand that this is not the correct place to report the error, but I know that developers are shared. Could you inform them?

thank you

**Comment by MatShir** [comment]
Hi @jlores, 
It is not the best place as you said to have support for checkout module.
They test it just now ! They didn't manage to reproduce it :/
I highly recommend asking support in the Tab Help in the module configuration to take a deeper look.
Have a good day !
